python/sdssdb/utils/ingest.py

```diff
@@ -15,11 +15,17 @@
 import os
 import re
 import warnings
+import inflect
```

Comment:
Import out of order and in incorrect block.

Reply:
done